Src/Support/Google.Apis.Auth/OAuth2/DefaultCredentialProvider.cs

```diff
@@ -263,7 +263,10 @@ private static UserCredential CreateUserCredentialFromParameters(JsonCredentialP
             }
             var initializer = new ServiceAccountCredential.Initializer(credentialParameters.ClientEmail);
             initializer.ProjectId = credentialParameters.ProjectId;
-            return new ServiceAccountCredential(initializer.FromPrivateKey(credentialParameters.PrivateKey));
+            return new ServiceAccountCredential(initializer.FromPrivateKey(credentialParameters.PrivateKey))
```

Comment:
Here I'd like to see the QuotaProject in the initializer, as the mutable part.

Reply:
Yes, will do.